src/main/java/com/study/cheeper/cheep/CheepRepository.java

```diff
@@ -7,10 +7,12 @@
 import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.data.repository.query.Param;
 
+import java.util.List;
+
 public interface CheepRepository extends PagingAndSortingRepository<Cheep, Long> {
     
     Page<Cheep> findByProfileId(Integer id, Pageable pageable);
 
-    @Query(value = "select u.name as name, u.email as email, c.id as cheepId, c.message as message, c.creation as creation from user u inner join relationship r on u.id = r.followed_id inner join cheep c on c.profile_id = r.followed_id where r.follower_id = :id", nativeQuery = true)
-    Page<TimelineProjection> allCheepsWhomIFollow(@Param("id") int id, Pageable pageable);
+    @Query(value = "select u.name as name, u.email as email, c.id as cheepId, c.message as message, c.creation as creation from user u inner join cheep c on c.profile_id = u.id left join relationship r on r.followed_id = u.id where r.follower_id = :id UNION ALL select u.name as name, u.email as email,c.id as cheepId, c.message as message, c.creation as creation from user u inner join cheep c on c.profile_id = u.id where u.id = :id", nativeQuery = true)
```

Comment:
Esse commit está sem paginação nos cheeps porque deu erro no UNION com Page:
SQLSyntaxErrorException: Unknown column 'u' in 'field list'

Vou consertar para os próximos commits, mas estou deixando de histórico